bucket/nmap.json

```
@@ -4,19 +4,19 @@
     "version": "7.80",
     "license": {
         "identifier": "GPL-2.0-only",
-        "url": "https://github.com/nmap/nmap/blob/master/COPYING"
+        "url": "https://svn.nmap.org/nmap/COPYING"
     },
     "url": "https://nmap.org/dist/nmap-7.80-setup.exe",
     "hash": "3b4d726bd366e7439367fa78a186dfa9b641d3b2ad354fd915581b6567480f94",
+    "pre_install": [
+        "Move-Item \"$dir\\npcap*.exe\" \"$dir\\npcap.exe\"",
+        "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\", \"$dir\\vcredist*\" -Recurse -Force"
+    ],
     "installer": {
-        "args": [
-            "/S",
-            "/D=$dir"
-        ]
+        "script": "Invoke-ExternalCommand -FilePath \"$dir\\npcap.exe\" -ArgumentList \"/S /D=$dir\\npcap\" -RunAs | Out-Null"
```

Unescaped dir after `/D`

```suggestion
        "script": "Invoke-ExternalCommand -FilePath \"$dir\\npcap.exe\" -ArgumentList \"/S /D=\"\"$dir\\npcap\"\"\" -RunAs | Out-Null"
```

Try installation with single quotes

---

It doesn't work with quotes.

---

nsis requires `/D$dir`. It breaks with quotes. See https://github.com/lukesampson/scoop/pull/2003